config.lua

```diff
@@ -27,3 +27,12 @@ end
 
 -- Show stairs/slabs/panels/microblocks in creative inventory (true or false):
 setting("bool", "stairsplus_in_creative_inventory", false)
+
+-- Enable customizing the tool repair value
+local tool_wear_default = -0.02
```

Comment:
I think this line can be removed as the comment below already documents the "default" value.